easybuild/easyconfigs/a/AlphaPulldown/AlphaPulldown-0.30.4-foss-2020b.eb

```diff
@@ -0,0 +1,93 @@
+# created by Denis Kristak (Inuits)
+easyblock = "PythonBundle"
+
+name = "AlphaPulldown"
+version = "0.30.4"
+
+homepage = "https://github.com/KosinskiLab/AlphaPulldown"
+description = """AlphaPulldown is a Python package that streamlines protein-protein
+interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ("OpenMM", "7.5.0"),
+    ("Kalign", "3.3.1"),
+    ('PyYAML', '5.3.1'),
+    ("cctbx-base", "2022.8"),
+    ('jax', '0.2.19'),  # also provides absl-py
+    ('Biopython', '1.78'),
+    ('h5py', '3.1.0'),
+    ('IPython', '7.18.1'),
+    ('JupyterLab', '2.2.8'),
+    ('matplotlib', '3.3.3'),
+    ('TensorFlow', '2.4.1'),
+    ('tqdm', '4.56.2'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('py3Dmol', '2.0.1.post1', {
+        'modulename': 'py3Dmol',
+        'checksums': ['add70ecf8f647970925eb8c1043c5c1343813fa49e613b77f0628e522c4148ac'],
+    }),
+    ('PDBFixer', '1.7', {
+        'source_urls': ['https://github.com/openmm/pdbfixer/archive/refs/tags/'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['a0bef3c52a7bbe69a6aea5333f51f3e7d158339be5829aed19b0344bd66d4eea'],
+    }),
+    ('dm-haiku', '0.0.4', {
+        'modulename': 'haiku',
+        'source_urls': ['https://github.com/deepmind/dm-haiku/archive/refs/tags/'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['e9896d161938b53f869fde207f3f6fca496d09b2a47d21dd9b6b65e897ad6aab'],
+    }),
+    ('toolz', '0.11.1', {
+        'checksums': ['c7a47921f07822fe534fb1c01c9931ab335a4390c782bd28c6bcc7c2f71f3fbf'],
+    }),
+    ('dm-tree', '0.1.6', {
```

Comment:
We have `dm-tree-0.1.5-GCCcore-10.2.0.eb`, so we should use that as a proper dependency?

Reply:
resolved